ynr/apps/popolo/migrations/0036_auto_20210310_1210.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.16 on 2021-03-10 12:10
```

Comment:
Could you rename this to something meaningful? We have so many migrations that I find good filenames are useful - I sometimes forget this though! `0036_add_party_name_and_description.py` or something?